src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100) + 1;
```

Comment:
100 is a magic number - lets add constant

Reply:
Done